src/coreclr/vm/threadsuspend.cpp

```diff
@@ -3711,17 +3710,27 @@ int RedirectedThrowControlExceptionFilter(
     // Resume execution at point where thread was originally redirected
     return (EXCEPTION_CONTINUE_EXECUTION);
 }
-#endif
-#endif // !TARGET_UNIX
+
+void RedirectedThrowControl()
```

Comment:
Extracted to separate method to make the compiler happy on win-x86+funclets. The `GCX_COOP` macro in the `#ifdef FEATURE_EH_FUNCLETS` puts some destructors on the stack and MSVC dislikes combining that with the inner SEH block.